lib/constants.dart

```diff
@@ -27,6 +27,7 @@ const queryCounties = "counties";
 const queryParishes = "parishes";
 const querySort = "sort";
 const queryOrder = "order";
+const queryId = "ids[]";
```

Comment:
Is this what you want? Just to make sure.

Reply:
Yup, it is.

And from the endpoints, you see that I used `_dio.getUri`.

This is because if we have an array of ids [1, 2, 3], the request query parameters should be `ids[]=1&ids[]=2&ids[]=3`